### ISSUE TITLE
error when trying out twdeps

### ISSUE THREAD

**Issue by clstaudt** [nerab/twdeps#9]
After installation with "gem install twdeps", the example call from the documentation failed with:

```
cls ~/Downloads $ task export rc.json.array=on rc.verbose=nothing | twdeps > deps.png
/usr/local/Cellar/ruby/1.9.3-p194/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require': cannot load such file -- bundler (LoadError)
    from /usr/local/Cellar/ruby/1.9.3-p194/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
    from /usr/local/Cellar/ruby/1.9.3-p194/lib/ruby/gems/1.9.1/gems/twdeps-0.0.3/bin/twdeps:3:in `<top (required)>'
    from /usr/local/Cellar/ruby/1.9.3-p194/bin/twdeps:23:in `load'
    from /usr/local/Cellar/ruby/1.9.3-p194/bin/twdeps:23:in `<main>'
```

**Comment by haniawni** [comment]
Hello!
I was getting the same error!
I'm on Ruby 2.0.0 from Mac OSx 10.11.6 prebaked ruby install (but with $GEM_HOME set somewhere I can actually modify)

fixed it by actually installing bundler:
`gem install bundler`

sent a pull request to include bundler in the dependency list, cause I only found this due to a fresh reinstall yesterday.

**Comment by nerab** [comment]
Fixed with #10
